editor.planx.uk/src/pages/FlowEditor/components/PreviewBrowser.tsx

```diff
@@ -163,6 +171,19 @@ const PreviewBrowser: React.FC<{
             </Link>
           </Tooltip>
 
+          {!!flowMetabaseLink && (
+            <Tooltip arrow title="Open analytics page">
```

Comment:
nit: I personally think the unpublished & published icons should stay next to each other at the far right since these are closely related actions. Could the analytics icon shift one left in position on this row, so it's directly to the right of "Toggle debug console" instead ?

Reply:
Agree with Jess' point about maintaining the grouping of the two right-side related icons and placing the analytics one in the middle.

However I think there is a value in maintaining all of the icons in place, and displaying more of an active/inactive state, rather than showing/hiding the icon.

Something like this (example has the inactive icon at 40% opacity):
<img width="608" alt="image" src="https://github.com/theopensystemslab/planx-new/assets/51156018/e444386f-57de-408c-85e8-91dcc9d8a411">

The logic being that is is easier/quicker to decipher whether an element is active/inactive through contrast than to work out if it is present or not.

Reply:
That's a great shout @ianjon3s I'll get this updated.

Reply:
I've got two ways of adding this but I'm a little unsure which option has the best balance for accessibility etc. 🤔 

I think it would make sense to have the icon as a `Link` although there's not really a concept of a link you can't click? Although there's is a concept of a `disabled` button. 

I was thinking of two ways of potentially indicating that the analytics page is unavailable:

Option 1:

Show the icon with a reduced opacity 

```JavaScript
<Tooltip arrow title="Analytics page unavailable">
  <BarChart style={{ opacity: 0.4 }} />
</Tooltip>
```

Option 2:

Return the material UI `Link` as a disabled button.

```JavaScript
<Tooltip arrow title="Analytics page unavailable">
  <Link
    component={"button"}
    disabled
  >
     <BarChart />
  </Link>
 
</Tooltip>
```

I think Option 2 is maybe better as a disabled button has a consistent style and there's precedent for it not having any action when clicked on? Whereas Option 1 is a random icon which doesn't convey much to those using screen readers etc.?

Reply:
I've got `Option 2` on this commit: b4b7e6602f79437443732c785ade50113ea44dfb

Reply:
I think the second option with `aria-disabled="true"` in place of the native `disabled` state would be ideal, in that it would allow for focus on the element, and therefore readability of the label for a screen reader.

Reply:
I tried swapping out `disabled` for `aria-disabled="true"` directly but it changes the styling of the icon to being blue rather than opaque. Although adding it to the component I think maybe has the best of both worlds? 

```JavaScript
        <Tooltip arrow title="Analytics page unavailable">
              <Link component={"button"} disabled aria-disabled={true}>
                <BarChart />
              </Link>
            </Tooltip>
```

Reply:
Agreed @Mike-Heneghan , worth adding in addition 👍

As a more general comment, the editor side of the product has neither been built or scrutinised at the same level as the public interface from an accessibility perspective, so I think a wider review is in order (and of course keeping it in the forefront when rethinking/design the interface or adding further features).